config/passport.js

```diff
@@ -0,0 +1,84 @@
+var GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
+var LocalStrategy = require('passport-local').Strategy;
+var BearerStrategy = require('passport-http-bearer').Strategy;
+var configAuth = require('./settings');
+var connection = require('./database');
+
+module.exports = function(passport) {
+
+	passport.serializeUser(function(user, done){
```

Comment:
Since the line is inside a function block, I decided to indent it. I'm not sure if the indentation is required for this particular way of defining the function. I shall unindent it if it does not follow standards.